multidim-arrays/bigarray_test.ml

```diff
@@ -3,17 +3,28 @@
  * In this example, tensors are represented as arrays of arrays of arrays.
  *)
```

Comment:
Sorry for the late response. I have a tight deadline coming up. I will make the suggested changes soon.